### ISSUE TITLE
Gitflow process

### ISSUE THREAD

**Issue by sleepylee** [zinzem/survey#3]
We understand that the application is small. But given the testing context and there are actually multiple setups that need to be crafted from scratch, there is always a need for separating the work for better review purpose. I can see from your separated commits within 1 Pull Request, but that still, just saying, not so right to do in our opinion.

**Comment by zinzem** [comment]
To me, code review is difficult when commits are too big and/or don't explain what they're about. I tried to avoid those two points. Splitting in several PR's would reflect more real life, but would not ease much code review, as far as I can see.

**Comment by sleepylee** [comment]
I totally agree on the point where a big push won't be effective for collaboration, but to our point, code review via PR is still, much easier than nothing like just 1 big merge :-)
